mozilla_django_oidc_db/utils.py

```diff
@@ -51,3 +56,25 @@ def extract_content_type(ct_header: str) -> str:
     content_type, _ = _parse_content_type_header(ct_header)
     # discard the params, we only want the content type itself
     return content_type
+
+
+def do_op_logout(config: OpenIDConnectConfigBase, id_token: str) -> None:
```

Comment:
Is this supposed to be implemented as a view by the projects that use this library? Maybe we could provide a default logout view?

I suppose it would also be good to mention how to use this in the docs

Reply:
I thought about it and ended up deliberately not documenting it because I find it shady... If I read the OpenID specification correctly, the intend is that the user agent (browser) of the user is redirected to this URL where they get logged out/need to confirm the log out.

I'm highly suspicious about keycloak that this also works for logging them out via a server-to-server call without user interaction. It is convenient for us, but I'm not sure it's compliant with the specification and if other OpenID Provider implementations will *also* work. So I'd like to keep this quiet ;)

Reply:
I've added a warning to the docstring, since the helper is picked up by autodoc anyway. I think this trade-off is okay.